cosmpy/aerial/client/__init__.py

```diff
@@ -22,25 +22,28 @@
 import json
 import math
 import time
-from dataclasses import dataclass
-from datetime import datetime, timedelta, timezone
+from datetime import datetime, timedelta
 from decimal import Decimal
 from typing import Any, Dict, List, Optional, Tuple
 
 import certifi
 import grpc
 from dateutil.parser import isoparse
-from google.protobuf.timestamp_pb2 import Timestamp
```

Comment:
Apparently adding this line back prevents protobuf descriptor error
`ImportError: cannot import name 'descriptor' from 'google.protobuf'`